template/package.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
Without the package there is an error during the CLI first run.

Reply:
Maybe can you confirm why that happens?

Also we should probably have a README to welcome folks